be/src/env/env.h

```diff
@@ -204,74 +199,84 @@ struct FilePathDesc {
     TStorageMedium::type storage_medium = TStorageMedium::HDD;
     std::string filepath;
     std::string remote_path;
+    std::string storage_name;
     std::string debug_string() const {
         std::stringstream ss;
-        ss << "local_path: " << filepath;
+        ss << "storage_medium: " << to_string(storage_medium) << ", local_path: " << filepath;
         if (!remote_path.empty()) {
-            ss << ", remote_path: " << remote_path;
+            ss << ", storage_name: " << storage_name << ", remote_path: " << remote_path;
         }
         return ss.str();
     }
+    static bool is_remote(TStorageMedium::type checked_storage_medium) {
+        return checked_storage_medium == TStorageMedium::S3 || checked_storage_medium == TStorageMedium::REMOTE_CACHE;
```

Comment:
What is the difference between S3 and REMOTE_CACHE?
Why `REMOTE_CACHE` be considered as `remote`?

Reply:
REMOTE_CACHE is the local cache path for remote path, if a data_dir is REMOTE_CACHE, it means the tablet in it will be set as a remote path.